recipe/patch/update_setup_config.patch

```diff
@@ -102,6 +102,6 @@ index a348dba..db06192 100755
  install_requires = [
 -    epicscorelibs.version.abi_requires(),
 -    pvxslibs.version.abi_requires(),
-     'nose2>=0.8.0',
```

Comment:
This is probably reasonable regardless, but this seems independent of including the separate tests.

Reply:
This is actually a must - everything breaks otherwise. It doesn't respect the `--no-deps` for some reason.

Reply:
Sorry if I wasn't clear: I mean it makes sense to remove this, since as I understand we don't use these for runtime, but only (hypothetically) for tests.